src/router.js

```diff
@@ -0,0 +1,20 @@
+const express = require('express');
+const router = express.Router();
+
+router.get("/", (req, res) => {
+  res.send('<h1>You are in the homepage</h1>');
+})
+
+router.get("*", (req, res)=> {
+  console.log(req.url);
```

Comment:
Is this `console.log` needed in source control?

Reply:
log deleted!